app/models/application_record.rb

```diff
@@ -1,6 +1,8 @@
 class ApplicationRecord < ActiveRecord::Base
   self.abstract_class = true
 
+  before_save :strip_attributes
```

Comment:
Should this be called in the ApplicationRecord model or in the relevant classes which inherit from it (e.g. `Vacancy`) ?

Reply:
I'm not sure, but I guess you could do a little test to check? 

We have other callbacks defined in this file (I think that's what they're called). 

`after_create { EventContext.trigger_event(:entity_created, event_data) }` for eg.

Perhaps you could put a breakpoint in the `trigger_event` method and then create an entity (a vacancy for eg) and see if you hit the breakpoint. If you do, I guess you can just put the before_save in this file.

Reply:
It does work when placed in here (have tested), but was wondering if it is costly to be calling it on all models (rather than just the ones that have user input and might have whitespace)